www/app/login/login.controller.js

```diff
@@ -5,14 +5,13 @@
     .module('budgetrentacar.login')
     .controller('LoginController', LoginController);
 
-    LoginController.$inject = ['$scope', '$state', '$ionicPopup'];
+    LoginController.$inject = ['$scope', '$state', '$ionicPopup','loginFirebaseService'];
 
-    function LoginController($scope, $state, $ionicPopup) {
-      var ctrlScope = $scope;
-      ctrlScope.user = { };
-      //screen.lockOrientation('portrait');
+    function LoginController($scope, $state, $ionicPopup, loginFirebaseService) {
+      var vm = $scope;
```

Comment:
Doing this:
```
var vm = $scope;
vm.something = stuff;
```
Is not different from doing this:
```
$scope.something = stuff;
```
Still a victim of ionic 1.7.12, issue #2499 ?